vcf2fhir/fhir_helper.py

```diff
@@ -488,72 +486,32 @@ def generate_final_json(self):
                 fhirReport['derivedFrom'] = derivedFrom
 
         for k, i in enumerate(od['contained']):
+            l2 = ["system", "code"]

```

Comment:
If you want you can declare this at the top of the file as constant and rename as `CATEGORY_ORDER`

Reply:
Please can you suggest a small name as CATEGORY_ORDER is giving character limit error